### PR TITLE
Add MongoDB connection URL to template message

### PR DESCRIPTION
**Pull request by spadgett**
@bparees PTAL

### CODE REVIEW

File: examples/db-templates/mongodb-ephemeral-template.json [hunk 1]
```diff
@@ -214,5 +214,6 @@
   ],
   "labels": {
     "template": "mongodb-ephemeral-template"
-  }
+  },
+  "message": "You can connect to the database using MongoDB connection URL mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@${DATABASE_SERVICE_NAME}/${MONGODB_DATABASE}"
```
Comment (bparees) [inline examples/db-templates/mongodb-ephemeral-template.json]: are we sure this url format is generic to all frameworks?  eg on eap i'd expect a jdbc:// url.

Comment (spadgett) [inline examples/db-templates/mongodb-ephemeral-template.json]: https://docs.mongodb.com/manual/reference/connection-string/#standard-connection-string-format

> The format is the same for all official MongoDB drivers.

But maybe we should update the message to qualify it?

Comment (bparees) [inline examples/db-templates/mongodb-ephemeral-template.json]: "the mongodb connection url is probably $FOO"
:)

Comment (spadgett) [inline examples/db-templates/mongodb-ephemeral-template.json]: I think it does always work? For example, in Java:

http://api.mongodb.com/java/2.10.1/com/mongodb/MongoClientURI.html

Comment (bparees) [inline examples/db-templates/mongodb-ephemeral-template.json]: ok

**Comment by bparees** [comment]
[merge]

**Pull request merged**